core/src/main/scala/sttp/tapir/FileRange.scala

```diff
@@ -3,7 +3,14 @@ package sttp.tapir
 import sttp.model.ContentRangeUnits
 import sttp.model.headers.ContentRange
 
-case class FileRange(file: TapirFile, range: Option[RangeValue] = None)
+import java.io.InputStream
+import java.net.URL
+
+sealed trait StaticResource {
```

Comment:
quite minor but:
1. the file needs renaming
2. are we modelling a static resource here? since we're dealing with arbitratary input streams, this can be anything really. So I'd consider two things: do we need to capture `FileRange` & `InputStreamRange` in a single hierarchy in the first place? And if so, what should be their common name

Sorry for the nitpicks here, but once this goes into `core`, there's no turning back ;)

Reply:
Yes in that case that would be best

Reply:
Actually, what if we move it to the `files` module entirely? Together with schema, codec, etc, this would make `core` totally free of files-related stuff. wdyt?

Reply:
Well this has become more than just files, as the new raw body type represents a lazily-evaluate input stream. The way it should have been done from the start ;) So I think this belongs to core